src/app/feature/consultant/services/consultant-store/consultant-store.service.ts

```diff
@@ -11,19 +11,19 @@ import { Consultant } from '@core/models';
 export const blankConsultant: Consultant = {
   id: null,
   urlProfileImage: null,
-  firstName: 'First',
-  lastName: 'Last',
-  title: 'Title',
-  practice: 'Practice',
-  email: 'Email',
+  firstName: '',
+  lastName: '',
+  title: '',
+  practice: '',
+  email: '',
   status: null,
   address: {
       lineOne: '10900 Stonelake Blvd. Suite 195',
       city: 'Austin',
       state: 'TX',
       zipCode: 78759
   },
-  phone: 1234567890,
+  phone: null,
```

Comment:
This causes the phone pipe to break. Id suggest adding a null check in the phone pipe rather than just changing this back.